backend/src/models/v2/Release.ts

```diff
@@ -17,13 +17,21 @@ export interface ReleaseInterface {
   fileIds: Array<string>
   images: Array<ImageRef>
 
+  comments: Array<ReviewComment>
+
   deleted: boolean
 
   createdBy: string
   createdAt: Date
   updatedAt: Date
 }
 
+export interface ReviewComment {
+  comment: string
```

Comment:
Can we rename this to something like `message`.  It's generally not a good thing to have both the object named the same as an attribute on the object.